File: output/ChatGPT/codenet/Java/C++/s897176066.cpp
#include <iostream>
#include <algorithm>
using namespace std;

int main() {
  int N;
  cin >> N;
  long A[N];
  for (int i = 0; i < N; i++) {
    cin >> A[i];
  }
  sort(A, A + N);
  for (int j = 1; j < N; j++) {
    if (A[j] == A[j-1]) {
      cout << "NO" << endl;
      return 0;
    }
  }
  cout << "YES" << endl;

}

File: output/CodeGen/codenet/Java/Python/s897176066.py
import java.util.*;

def main():
    N = int(input())
    A = [int(x) for x in input().split()]
    A.sort()
    for j in range(1, N):
        if A[j] == A[j - 1]:
            print("NO")
            break
    print("YES")

if __name__ == "__main__":
    main()